contracts/ParsecBridge.sol

```diff
@@ -522,4 +537,148 @@ contract ParsecBridge {
     exitable_at = priority >> 128;
   }
 
+  function startInflightExit(bytes32[] _tx, bytes32[] _i1Proof, bytes32[] _i2Proof, uint16 _oindex) public {
+
+//  Honest owners of outputs to t “piggyback” on this exit and post a bond
+//  Users who do not piggyback will not be able to exit.
+//  In case of doublespending user couldn't post a bond due to possible challenge
+
+    //check that txHash has not been used in inflight exit
+    bytes32 txHash;
+    bytes memory txData;
+//    ( ,txHash, txData) = TxLib.validateProof(txData); //todo parse txData for inflight exit in parsec-lib
+    require(inflightExits[txHash].txHash == bytes32(0));
+
+    TxLib.Tx memory tx = TxLib.parseTx(txData);
+
+    _validateInputs(tx.ins, _i1Proof, _i2Proof);
+
+    // todo validate tx sig
+
+    //place bond for output
+    ERC20(tokens[0].addr).transferFrom(msg.sender, address(this), inflightBond); //currently set psc tokens
+
+    //create exit with bond
+    InflightExit storage exit = inflightExits[txHash];
```

Comment:
`inflightExits[txHash] = InflightExit({key: value, ...})` <- why not like this?
or should be we able to update existing exits?

Reply:
Can't copy TxLib.Input memory[] memory to storage, thats why.
Btw, from Solidity docs:
`If the storage struct has tightly packed properties, initialize it with separate assignments: x.a = 1; x.b = 2;. In this way it will be easier for the optimizer to update storage in one go, thus making assignment cheaper.`